README.md

```diff
@@ -19,6 +19,18 @@ Just add the library dependency to your app's `build.gradle`:
 compile 'com.schibsted.spain:prado:1.0.0'
 ```
 
+Then you have to declare which image provider do you want to use:
+- Picasso
+```
+compile 'com.squareup.picasso:picasso:2.5.2'
+compile 'com.jakewharton.picasso:picasso2-okhttp3-downloader:1.1.0'
```

Comment:
Yes because we use okhttp3 as picasso downloader, if you don't specify one picasso uses it's own. By using and cofiguring okhttp3 we are able to use its internal cache, so the performance is increased a lot!